pyaerocom/config.py

```diff
@@ -243,6 +204,24 @@ def __init__(self, config_file=None, try_infer_environment=True):
         self.last_config_file = None
         self._ebas_flag_info = None
 
+        self._outhomename = "MyPyaerocom"
+
+        with resources.path("pyaerocom.data", "paths.ini") as path:
+            self._config_ini_lustre = str(path)
+        with resources.path("pyaerocom.data", "variables.ini") as path:
+            self._var_info_file = str(path)
+        with resources.path("pyaerocom.data", "coords.ini") as path:
+            self._coords_info_file = str(path)
+
+        self._user = getpass.getuser()
+        self.my_pyaerocom_dir = os.path.join(f"{os.path.expanduser('~')}", self._outhomename)
+
+        self.DONOTCACHEFILE = None
```

Comment:
I'd opt for `DO_NOT_CACHE_FILE` for readability

Reply:
only used in tests, but done anyway